addons/mail/static/src/components/composer/composer_tests.js

```diff
@@ -1842,6 +1842,22 @@ QUnit.test('remove an uploading attachment aborts upload', async function (asser
     });
 });
 
+QUnit.test("basic rendering when sending a message to the followers and thread doesn't have a name", async function (assert) {
+    assert.expect(1);
+
+    await this.start();
+    const thread = this.env.models['mail.thread'].create({
+        composer: [['create', { isLog: false }]],
+        id: 20,
+        model: 'res.partner',
+    });
+    await this.createComposerComponent(thread.composer, { hasFollowers: true });
+    assert.strictEqual(
+        document.querySelector('.o_Composer_followers').textContent.trim(),
+        "To: Followers of  this document"
+    );
```

Comment:
Weird the double white space between _"of"_ and _"this document"_.
Underlying cause is single sentence being inappropriately split in template with tags.

Don't fix the problem in template, because this will delay the fix further, and we are aware of such problems in template. I would just replace all whitespaces by no whitespaces. Assertion string is a bit ugly, but it's best than the double arbitrary whitespace.

Also don't forget assertion comment.